### ISSUE TITLE
Release current master to PyPI

### ISSUE THREAD

**Issue by akhayyat** [yourlabs/django-autocomplete-light#1316]
First of all, thank for a great project!

Would it be possibles to update the release available on PyPI?

The current release (`3.9.4`) is suffering from this error when using `Select2GroupQuerySetView`:

```
.../dal_select2/views.py", line 105, in get_results
    groups = collections.OrderedDict()
NameError: name 'collections' is not defined
```
Even the latest pre-release on PyPI (`3.9.5rc1`) suffers from the same problem:

https://github.com/yourlabs/django-autocomplete-light/compare/3.9.5-rc1...master#diff-45f913a82e75bc406f5f040fd71c0df7f7319b41d0ae0602a69581b0abaccdc3L114

which seems to be fixed in `master`:

https://github.com/yourlabs/django-autocomplete-light/compare/3.9.5-rc1...master#diff-45f913a82e75bc406f5f040fd71c0df7f7319b41d0ae0602a69581b0abaccdc3R109

**Comment by sveetch** [comment]
Yes it would be nice to get the new release, i'm facing issue with dark mode which i've seen has been resolved since 3.9.4 and so i need to patch it on my own in my project.

If a lovely maintainer comes here and see this, it would be very nice, thank you.

**Comment by jpic** [comment]
Thanks for the heads up! Will issue -rc releases after checking out from now on, given they are completely scripted now.

Please report if 3.9.5-rc6 works for you in which case I'll fire a 3.9.5 release right away

**Comment by jpic** [comment]
Anyone tried? It's the 11th birthday of the first commit today, would be nice to release 3.9.5 :P

**Comment by sveetch** [comment]
@jpic I'm very sorry for the delay to check your rc6. I found some to check for the 3.9.5-rc6 on my projects, it correctly fix my problem with the dark mode and all my unittest are passing, so from what i know the 3.9.5-rc6 is a success !

**Comment by jpic** [comment]
Thanks for your feedback!

Released 3.9.5, please reopen in case of problem
